exercises/react/example.cr

```diff
@@ -0,0 +1,69 @@
+module React
```

Comment:
Now in a module so that `InputCell` can call `ComputeCell#propagate` but outsiders can't. https://github.com/exercism/xcrystal/pull/44#discussion_r78289024 for context